src/main/resources/static/api.yml

```diff
@@ -37,7 +37,35 @@ paths:
         400:
           description: Bad input
 
+  /courses/csv:
```

Comment:
So if I'm understanding correctly, we can't have:
`get courses/` with `text/csv` returns the CSV
`get courses/` with `application/json` returns the json response

I wonder if we should therefore have all of these endpoints be `/csv`, both for consistency of endpoints _and_ if in future we were to add another `/offerings` endpoint that would fetch all offerings, this would cause a clash.

What do you think?

Reply:
There isn't a nice solution to this.  I don't object to adding the /csv suffix, but I'd rather the code generator did the right thing! Or... that I could work out how to get the code generator to do the right thing.

Reply:
Done.